.travis.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         # On pull requests, run all flake8 tests on modified code
         # if clause avoids `fatal: ambiguous argument 'origin/master'` on git push
         - if [ "$TRAVIS_EVENT_TYPE" == "pull_request" ]; then
-            git diff origin/master -U0 | flake8 --diff --max-line-length=88;
+            git diff origin/master -U0 | flake8 --diff --max-line-length=127;
```

Comment:
Let’s stick with 88 characters max per line. That is reasonable and will align with psf/black.